x/programs/wasmlanche/src/host.rs

```diff
@@ -147,6 +160,27 @@ mod test_wrappers {
                 None => HostPtr::null(),
             }
         }
+
+        pub fn put(&self, key: &[u8], value: Vec<u8>) {
+            self.state.borrow_mut().insert(key.into(), value);
+        }
+
+        pub fn deploy(&self) -> u8 {
+            assert!(
+                *self.deploys.borrow() < u8::MAX,
+                "Too many deploys, max is 255"
+            );
+            *self.deploys.borrow_mut() += 1;
+            *self.deploys.borrow()
+        }
```

Comment:
Are you reading anywhere or just preventing 256 deploys?

Reply:
just preventing overflow, but I remembered we have `overflow-checks=true` so i don't need this